sql/src/main/java/io/crate/planner/operators/FetchOrEval.java

```diff
@@ -488,13 +488,20 @@ private ExecutionPlan planWithEvalProjection(PlannerContext plannerContext,
                                                  List<Symbol> sourceOutputs,
                                                  Row params,
                                                  SubQueryResults subQueryResults) {
-        PositionalOrderBy orderBy = executionPlan.resultDescription().orderBy();
-        PositionalOrderBy newOrderBy = null;
         SubQueryAndParamBinder binder = new SubQueryAndParamBinder(params, subQueryResults);
         List<Symbol> boundOutputs = Lists2.map(outputs, binder);
-        if (orderBy != null) {
+        PositionalOrderBy orderBy = executionPlan.resultDescription().orderBy();
+        PositionalOrderBy newOrderBy;
```

Comment:
this could have the value of orderBy (or straight up null) and then the if statement would not require an else branch 

eg: 

```
PositionalOrderBy newOrderBy = null;

if (orderBy != null ) {
    newOrderBy = orderBy.tryMapToNewOutputs(sourceOutputs, boundOutputs);
    ...
}

Reply:
I prefer to keep local variables effectively final.